ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -67,6 +66,7 @@ StatusDialog {
         NumberAnimation { duration: 1000; easing.type: Easing.OutExpo; alwaysRunToEnd: true}
     }
 
+    onOpened: payPanel.tokenAmount = root.swapInputParamsForm.fromTokenAmount
```

Comment:
why wont the binding work?

Reply:
It does the endless binding loop; I'll create a video to demonstrate

Reply:
yes but then in that case not only token amount the rest need to be handled the same way. Either keep the binding loop for all variables and we fix it another task or do it here as well? cause this seems like fixing just one part of the same problem

Reply:
and ideally it would be better to assign in payPanel as 
tokenAmount: root.swapInputParamsForm.fromTokenAmount and simply stop assigning onValueChanged back to root.swapInputParamsForm.fromTokenAmount as we were discussing  that could be the way to go.

Reply:
Yeah, we should have a separate issue for this logged, and tackle it there

Reply:
My idea is roughly to add this to `SwapModal`:
```qml
    QtObject {
        id: preparedParams
        readonly property string selectedPayTokenId: payPanel.tokenKey
        readonly property string selectedPayAmount: payPanel.tokenAmount
        // TODO add more
    }
```

and pass that to the backend whenever the user hits the "Swap/Approve" button; wdyt @Khushboo-dev-cpp

Reply:
And then the `SwapInputParamsForm swapInputParamsForm` can truly serve its purpose namely, to act as initial input only

Reply:
@caybro yes with all the binding loops you've noticed it makes sense to do that. I think we don't need a separate QTObject to do that though, we can simple call payPanel.tokenAmount. receivePanel.tokenAmount etc directly instead.

Reply:
Yeah, I think it's really better to stop assigning the UI values back to the `SwapInputParamsForm swapInputParamsForm`, makes no sense imho

Reply:
That said, I can get rid of that here in this PR; this will require changing quite a lot of tests but it's worth it imho